algorithms/hhl/hhl/hhl.ipynb

```diff
@@ -1,982 +1,1292 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        qmod_hhl = create_model(main)
</pre><p>BTW, you can do </p><p> </p>

<pre class="ql-syntax" spellcheck="false">qmod_hhl = create_model(main, execution_preferences=ExecutionPreferences(
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; num_shots=1, backend_preferences=backend_preferences
&nbsp; &nbsp; &nbsp; &nbsp; ))
</pre><p>without using <code>set_execution_preferences</code> . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>